src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
+    private UserResponseMapper userResponseMapper;
```

Comment:
```suggestion
    private DtoResponseMapper<UserResponseDto, User> userResponseMapper;
```
we need to rely on abstraction, not implementation